### DISCUSSION TITLE
Why can't the video in Firefox be recognized?

### DISCUSSION THREAD

**Discussion by Unlimitedtang** [aclap-dev/video-downloadhelper#1257] [Help and Q&A]
### Your question

Version: 9.1.0.44 （Firefox ）
URL：https://www.youtube.com/shorts/guCthjETfcY 

I was able to download this link before, but now I can't recognize the video when I open it? And I can't download it.The icon is also grey

### Addon & CoApp version and details

Version: 9.1.0.44 （Firefox ）
URL：https://www.youtube.com/shorts/guCthjETfcY 

I was able to download this link before, but now I can't recognize the video when I open it? And I can't download it.The icon is also grey

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

I am running Windows 11 Pro 64-bit version 23H2 build 22631.4169, Firefox 130.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.  That would be complete information.  You should post complete information for your setup.  The easy way would be to click the button in the VDH settings that says `Copy information to clipboard`, then paste that in here.  But you can also do a shortened version following the model I've just shown you for my system.

With my setup, I am seeing this:

![#01](https://github.com/user-attachments/assets/66863185-b44f-4dca-afba-b5bad4f91cfd)

Doesn't look like a problem to me.

**Reply by 5EJKER** [reply]
hey I have a question?

do you still have this version installed on firefox

video download helper

[9.1.0.44](http://9.1.0.44/)

because now the creator has removed the versions that worked for free and left only the old 7.0

9.1.0.44-generally there is probably no better plugin for downloading movies. it finds everything it can practically always

but recently when I transferred the profile from browser to browser

it unfortunately

something probably overwrote it

and it updated

if you have this version

could you send me the file for it. it should be called like that. "(b9db16a4-6edc-47ec-a1f4-b86292ed211d).xpi" 

and it is in the %appdata% location and there in firefox in profiles --> extension 

I also found for chrome someone saved this version so I can send it to you. but for firefox it is nowhere. 

https://www.transfernow.net/dl/Videodowloaderdlachrome

**Comment by Unlimitedtang** [comment]
I am in China and need to use a VPN, which may have caused the failure. Is there any stable VPN that can be used?




这个名字够长长长长长长长长长长吗
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "aclap-dev/video-downloadhelper"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月13日(星期五) 中午12:12
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [aclap-dev/video-downloadhelper] Why can't the video in Firefox be recognized? (Discussion #1257)





 
I do not work for VDH.  I am just another user.
 
I am running Windows 11 Pro 64-bit version 23H2 build 22631.4169, Firefox 130.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.  That would be complete information.  You should post complete information for your setup.  The easy way would be to click the button in the VDH settings that says Copy information to clipboard, then paste that in here.  But you can also do a shortened version following the model I've just shown you for my system.
 
With my setup, I am seeing this:
 
01.png (view on web)
 
Doesn't look like a problem to me.

**Comment by 5EJKER** [comment]
hey I have a question?

do you still have this version installed on firefox

video download helper

[9.1.0.44](http://9.1.0.44/)

because now the creator has removed the versions that worked for free and left only the old 7.0

9.1.0.44-generally there is probably no better plugin for downloading movies. it finds everything it can practically always

but recently when I transferred the profile from browser to browser

it unfortunately

something probably overwrote it

and it updated

if you have this version

could you send me the file for it. it should be called like that. "(b9db16a4-6edc-47ec-a1f4-b86292ed211d).xpi" 

and it is in the %appdata% location and there in firefox in profiles --> extension 

I also found for chrome someone saved this version so I can send it to you. but for firefox it is nowhere. 

https://www.transfernow.net/dl/Videodowloaderdlachrome
